### DISCUSSION TITLE
How to determine the value of captureSize?

### DISCUSSION THREAD

**Discussion by h170607** [PoisonAlien/maftools#978] [General]
hi,
I am using the tmb() function to calculate the TMB value of a set of panel sequencing data:
`tmb1 <- tmb(maf, captureSize =1.869664, logScale = T)`
Where 1.869664 is the length of the targeted region calculated from the bed file.
Then most of the TMB values obtained are above 100, which are suspicious from experience.
Is the value of captureSize wrong?

**Comment by PoisonAlien** [comment]
Hi.

Thanks for the issue. If you have estimated the size from the BED file, it is the right way. Note that the capture size should in MBs.
I am not sure why the tmb is so high, but you should check your variant calling and filtering steps.

**Comment by h170607** [comment]
Hi,
My pipeline is like this:
`gatk Mutect2 -L $bed -R $reference -I normal.bam -I tumor.bam -normal $normal -tumor $tumor -O ${sample}.raw.vcf`

`gatk FilterMutectCalls -R $reference -V ${sample}.raw.vcf -O ${sample}.Filter.vcf`

`convert2annovar.pl --format vcf4 --includeinfo ${sample}.Filter.vcf > ${sample}.avinput --allsample --withfreq`

`table_annovar.pl ${sample}.avinput $annovar/humandb/ -buildver hg19 -out ${sample} -protocol refGene,clinvar_20200316,cosmic91_coding,cosmic91_noncoding,exac03,1000g2015aug_eas,1000g2015aug_all,esp6500siv2_all,avsnp150,gnomad_genome,gnomad_exome,mcap13,revel,dbnsfp35a,intervar_20180118 -operation g,f,f,f,f,f,f,f,f,f,f,f,f,f,f -csvout`

Then the resulted `${sample}.hg19_multianno.csv` was converted to the maf format using annovarToMaf(). Is there anything wrong with this process?
